### ISSUE TITLE
Challenges Submit button does nothing

### ISSUE THREAD

**Issue by jkaufman-LogRhythm** [CTFd/CTFd#1998]
**Environment**:

- CTFd Version/Commit: 3.3.1
- Operating System: Centos 7
- Web Browser and Version: Attempted on Chrome, Firefox & Edge

**What happened?**
When answering a challenge, clicking submit doesn't do anything

**What did you expect to happen?**
Answer should be submitted

**How to reproduce your issue**
So far I just have a `docker-compose up` success without issues
I add one challenge, log out and register as a new user to answer the challenge.
Clicking submit doesn't do anything!

The console shows no function being called at all.

ps - I also cannot access the db? I docker ps -a and see mariadb running but no port is associated with it.

**Any associated stack traces or error logs**
If I come across some I will, sadly compile time of docker compose up shows no errors, when clicking the button no errors appear.

**Comment by ColdHeat** [comment]
That usually happens when the theme or plugin sends a request that CTFd doesn't expect. Does CTFd show any logs or a stack trace when you click submit?

**Comment by jkaufman-LogRhythm** [comment]
Sadly there is no stack trace. I did notice on another instance we have running we have a radio button questions
`POST https://subdomainName.Company.com/api/v1/challenges/attempt 500 (INTERNAL SERVER ERROR)

t.default	@	challenges.min.js?d=1e3de56f:1
e.request	@	challenges.min.js?d=1e3de56f:1
o.API.post_challenge_attempt	@	challenges.min.js?d=1e3de56f:1
challenge.submit	@	VM897:117
(anonymous)	@	challenges.min.js?d=1e3de56f:1
dispatch	@	vendor.bundle.min.js?d=1e3de56f:110
f.handle	@	vendor.bundle.min.js?d=1e3de56f:110

(headers)
Request URL: https://subdomainName.Company.com/api/v1/challenges/attempt
Request Payload:
{challenge_id: 13, submission: "TheRadioButtonAnswer"}
challenge_id: 13
submission: "TheRadioButtonAnswer"

`

**Comment by ColdHeat** [comment]
Are you running the latest version of the multiple choice plugin? It sounds like the multiple choice plugin might be an old version that didn't handle the situation when a user didn't make a selection. 

That being said, if you just click submit without making a choice in a multiple choice question, there's really not much way that you'd get the challenge right.

**Comment by jkaufman-LogRhythm** [comment]
You are right, it was the multiple choice. After updating to that plugins latest it appears to work. The inputs are failing but switching back to core (inputs standard that is) submit works, going to merge our hacker-theme with the latest version of that and we might be good to go. 

Thank you kindly for the fast response to this.

**Comment by the-src** [comment]
I encountered the same problem, I think it is somehow related to your browser's cookies. I reset the cookies and it started to work.
